apps/user-profile/src/assets/config/defaultConfig.json

```
@@ -8,7 +8,8 @@
   "instance_favicon": false,
   "auto_auth_redirect": true,
   "supported_languages": [
-    "en"
+    "en",
+    "cs"
```

It looks good otherwise.
```suggestion
    "en"
```

---

oops, leftover from testing, fixed